webservices/utils.py

```diff
@@ -141,14 +142,19 @@ def related(cls):
 )
 
 
-def document_description(report_year, report_type=None, document_type=None):
+def document_description(report_year, report_type=None, document_type=None, form_type=None):
     if report_type:
         clean = re.sub(r'\{[^)]*\}', '', report_type)
     elif document_type:
         clean = document_type
+    elif form_type and form_type in decoders.form_types:
+        clean = decoders.form_types[form_type]
     else:
         clean = 'Document '
-    return '{0}{1}'.format(clean, report_year)
+
+    if form_type and form_type == 'RFAI':
+       clean = "RFAI: " + clean
+    return re.sub("\s\s+" , " ", '{0} {1}'.format(clean, report_year))
```

Comment:
* Prefer raw strings for regular expressions: `re.sub(r'...', '...')`
* Prefer single quoted strings
* If the purpose of this is to replaces multiple spaces with a single space, we can use `re.sub(r'\s+', ' ', ...)`
* Else if the purpose is to replace two or more spaces, we can use `re.sub(r'\s{2,}', ...)`, although I'm guessing we want the previous version.
* But actually, if the only purpose of this regex is to remove trailing spaces after `clean`, we don't even need it: `return '{0} {1}'.format(clean.strip(), report_year)`.